packages/bull-vault/src/LeverageBull.sol

```diff
@@ -124,7 +133,7 @@ contract LeverageBull {
         _depositEthInEuler(ethToLend, true);
         _borrowUsdcFromEuler(usdcToBorrow);
 
-        return (ethToLend, usdcToBorrow);
+        return (ethToLend, usdcToBorrow, IEulerEToken(eToken).balanceOfUnderlying(address(this)));
```

Comment:
Could be there an issue of using this balanceOf()? either here or for calc wethToLend overall? e.g if someone transfer some amount right before leverage or full rebalance ?